components/create/CreateLanding.vue

```diff
@@ -49,18 +51,29 @@
               <p>{{ $t('mint.landing.massNftDesc') }}</p>
             </div>
           </div>
-        </nuxt-link>
+        </div>
       </div>
     </div>
   </div>
 </template>
 
 <script setup lang="ts">
 import { NeoIcon } from '@kodadot1/brick'
+import type { RawLocation } from 'vue-router/types/router'
 
+const instance = getCurrentInstance()
+const { doAfterLogin } = useDoAfterlogin(instance)
 const { urlPrefix } = usePrefix()
 const route = useRoute()
 const router = useRouter()
+
+const gotoPathAfterLogin = (path: RawLocation) => {
+  doAfterLogin({
+    onLoginSuccess: () => {
+      router.push(path)
```

Comment:
If it remember correctly we have some util "navigate To"
Is it suitable here?

Reply:
`navigateTo` works as the same